app/services/ethereum/bridge/service.go

```diff
@@ -77,7 +91,76 @@ func (bsc *BridgeContractService) SubmitSignatures(opts *bind.TransactOpts, ctm
 		signatures)
 }
 
+func (bsc *BridgeContractService) GetCustodians() []string {
+	return bsc.custodians.Get()
+}
+
+func (bsc *BridgeContractService) GetServiceFee() *big.Int {
+	return bsc.servicefee.Get()
+}
+
+func (bsc *BridgeContractService) UpdateServiceFee(opts *bind.CallOpts) {
+	newFee, err := bsc.contractInstance.ServiceFee(opts)
+	if err != nil {
+		log.Error("Failed to get service fee.", err)
```

Comment:
I think that if this fails, we will set invalid value to `serviceFee`. Maybe we should do `log.Fatal` since the initialisation of the contract variables will not be performed and we won't be able to run the validator node.